cm4-tool/tool.c

```diff
@@ -133,12 +133,12 @@ int main(int argc, char *argv[])
 		msg.o.size = 0;
 
 		if (file) {
-			i->type = m4_loadFile;
+			i->cm4_type = CM4_LOAD_FILE;
```

Comment:
Message will not be delivered because i->id is not set.

Reply:
Right, we need to pass id from lookup